packages/manager/apps/pci-volume-snapshot/src/pages/listing/useDatagridColumn.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  DataGridTextCell,
+  DatagridColumn,
+  useTranslatedMicroRegions,
+} from '@ovh-ux/manager-react-components';
+import { useTranslation } from 'react-i18next';
+import { useBytes } from '@ovh-ux/manager-pci-common';
+import { TVolumeSnapshot } from '@/api/hooks/useSnapshots';
+import { useFormattedDate } from '@/hooks/useFormattedDate';
+import StatusComponent from './Status.component';
+import ActionsComponent from './Actions.component';
+
+export const useDatagridColumn = () => {
```

Comment:
As I mentioned in previous µapps, this should not be a hook as, indeed, it's only a plain function that returns static columns consumed by one single component.

If the reason to create this hook is to only handle other hooks, we can consider given a formatter object as parameter:)

Reply:
It's complicated to transform this hook into a function because it uses the hook `useFormattedDate` for each item.
Because of that, this hook must be called inside the component otherwise React will complain about hooks called a different number of times.

Or, we must also convert `useFormattedDate` to a simple function but it uses `useMemo`.